assets/custom.js

```diff
@@ -10,8 +10,16 @@ function activate_sidebar() {
 function activate_dropdown() {
```

Comment:
what I understand from your code is, you are trying to assign any object to have dropdown, so remove function declaration lines instead.